capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Feature                                    | Amazon Web Services (AWS) | Microsoft Azure | Google Cloud Platform (GCP) | Red Hat OpenStack Platform | IBM PowerVC via the OpenStack API     | IBM Power Systems Virtual Servers |
+| ------------------------------------------ | ------------------------- | --------------- | --------------------------- | -------------------------- | ------------------------------------- | --------------------------------- |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Drift Comparison                  | Yes                       | Yes             | No                          | Yes                        | No                                    | No                                |
+| Track Instance Genealogy                   | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Capture Cloud Layer Timelines              | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Capture Instance Event Timelines           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Discovery - Provider                       | No                        | No              | No                          | No                         | No                                    | No                                |
+| Disk Addition to Instance                  | Yes                       | Yes             | No                          | Yes                        | No                                    | Yes                               |
+| Key Pairs Inventory                        | Yes                       | No              | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Key Pairs Management                       | Yes                       | No              | No                          | Yes                        | Yes                                   | Yes                               |
+| Reporting                                  | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
```

Comment:
Are we sure reporting wouldn't "just work"?

Reply:
Since the 'IBM PowerVC via the OpenStack API' column is marked `No` I'm assuming this includes reporting C&U information.